test/src/glossary/widgets/search_bar_test.dart

```diff
@@ -240,9 +240,10 @@ void main() {
        */
       // Init widget
       final l10n = await setupLocalizations();
+      var lastSearchTxt = "";
 
       await tester.pumpLocalizedWidget(GlossarySearchBar(
-        searchGlossary: (String searchText) => {},
+        searchGlossary: (String searchText) => {lastSearchTxt = searchText},
         filterGlossary: () => {},
```

Comment:
Dans Flutter pas besoin de `=>` j'approuve la PR pareil mais si tu veux bien ajouter la suggestion 😄 

```suggestion
        searchGlossary: (String searchText) {lastSearchTxt = searchText},
        filterGlossary: () {},
```